gpu-service/jobs.py

```
@@ -174,42 +197,55 @@ def extract_and_upsert_faces(
         return 0
 
     frame_rgb = cv2.cvtColor(frame_bgr, cv2.COLOR_BGR2RGB)
-    points: list[models.PointStruct] = []
+    
+    # Prepare batch data
+    embeddings = []
+    locations = []
+    metadatas = []
+    object_names = []
 
     for idx, (face_key, face_data) in enumerate(faces.items()):
```

Ruff (0.14.7)

Loop control variable `face_key` not used within loop body

Rename unused `face_key` to `_face_key`

(B007)